lib/tapioca/cli.rb

```diff
@@ -29,6 +32,16 @@ class Cli < Thor
                   banner: "gem:level",
                   desc: "Overrides for typed sigils for generated gem RBIs"
 
+    desc "init", "initalizes folder structure"
+    def init
+      create_file(Generator::SORBET_CONFIG, skip: true) do
+        "--dir\n."
+      end
+      create_file(Generator::DEFAULT_POSTREQUIRE, skip: true) do
+        "# Add your extra requires here"
```

Comment:
This should also be explained in the README, furthermore since it's not something *required* by the official `srb` tool.

Reply:
Indeed. I'll write a more detailed documentation after this PR.